build.gradle

```diff
@@ -428,7 +425,7 @@ bootWithCCD {
     environment 'CITIZEN_UPDATE_CASE_STATE_ENABLED', 'true'
     environment 'LOGGING_LEVEL_UK_GOV_HMCTS_DIVORCE', 'info'
     environment 'ROLE_ASSIGNMENT_URL', 'http://localhost:4096'
-    environment 'CASE_DATA_STORE_BASEURL', 'http://localhost:4452'
+ //   environment 'CASE_DATA_STORE_BASEURL', 'http://localhost:4452'
```

Comment:
Are you planning to leave in this change?

Reply:
oops, no, ta